src/field_theory/finite/basic.lean

```diff
@@ -131,7 +131,7 @@ variable (K)
 
 theorem card (p : ℕ) [char_p K p] : ∃ (n : ℕ+), nat.prime p ∧ q = p^(n : ℕ) :=
 begin
-  haveI hp : fact p.prime := char_p.char_is_prime K p,
+  haveI hp : fact p.prime := @char_p.char_is_prime K _ _ _ _ p _,
```

Comment:
Are you sure yo uneed the udnerscores here?

Reply:
I think that you are right: this seems to be unneeded!